src/components/PhotoViewer/PhotoViewer.tsx

```diff
@@ -0,0 +1,76 @@
+import "./PhotoViewer.scss";
+import React from "react";
+import { useState, useEffect } from "react";
+
+interface Photo {
+  id: number;
+  img_src: string;
+  earth_date: string;
+}
+
+interface RoverData {
+  photos: Photo[];
+}
+
+const PhotoViewer: React.FC = () => {
+  const [roverData, setRoverData] = useState<RoverData>();
+  const [selectedRover, setSelectedRover] = useState<string>("spirit");
+  const [coverImageUrl, setCoverImageUrl] = useState<string>("");
+  const [date, setDate] = useState<string>();
+  const URL = `https://api.nasa.gov/mars-photos/api/v1/rovers/${selectedRover}/photos?api_key=${process.env.REACT_APP_API_KEY}&sol=100`;
```

Comment:
nice work here! would you mind adding instructions to the README about this? it doesn't need to be too in depth but mostly the stating variable name and file (this is mostly for if new people would join the project but also good for when the others pull this change need to update their env file)

Reply:
updated README with an API key section